### DISCUSSION TITLE
Configure Ramp up time?

### DISCUSSION THREAD

**Discussion by batocoder** [abstracta/jmeter-dotnet-dsl#1] [Q&A] [answered]
Is this a supported feature? Can you point me to where I can add code for this? Thanks

**Comment by rabelenda** [comment]
Hello, thank you for asking about this. We will work on adding it. Stay tuned!

**Comment by rabelenda** [comment]
In general adding such support would only require modifying the test element class (in this case DslThreadGroup). But, this scenario actually requires some further changes for serialization and deserialization of associated properties and a little refactoring in existing code.

**Comment by rabelenda** [comment, marked as answer]
We have just released a [new version](https://github.com/abstracta/jmeter-dotnet-dsl/releases/tag/v0.3) which adds methods to specifies ramps & holds.
